src/clarity-angular/popover/common/_popover.clarity.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('popover.clarity') {
+    .is-off-screen {
+        position: fixed !important;
+        height: 1px !important;
+        width: 1px !important;
+        top: -1px !important;
+        padding: 0 !important;
+        margin: 0 0 -1px 0 !important;
+        border: none !important;
```

Comment:
`box-shadow`?

Reply:
i only added properties which occupy space. thats why ignored it. do you think we need to add `box-shadow`?

Reply:
My bad, I forgot that it was `visibility: hidden` so box-shadow wouldn't show. I was scared of it showing at the top of the page, not the scrollbars.